RadzenBlazorDemos/Pages/DataGridPagerApi.razor

```diff
@@ -38,7 +38,10 @@
 </RadzenStack>
 
 @code {
-    string pagingSummaryFormat = "Displaying page {0} of {1} <b>(total {2} records)</b>";
+    RenderFragment<PagingInformation> pagingSummaryTemplate = (info) =>@<span>
```

Comment:
Could you please define the template inline within RadzenDataGrid's configuration?